src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/models/rgw-multisite.ts

```diff
@@ -61,3 +61,8 @@ export enum FlowType {
   directional = 'directional',
   symmetrical = 'symmetrical'
 }
+
+export interface ZoneType {
```

Comment:
nitpick:
suggestion:
```suggestion
export interface Zone{
```